en/00_Introduction.adoc

```diff
@@ -4,96 +4,177 @@
+Although each chapter is written as a follow-up on the previous one, it is
+also possible to read the chapters as standalone articles introducing a
+certain Vulkan feature.
 That means that the site is also useful as a reference.
-All of the Vulkan functions and types are linked to the specification, so you can click them to learn more.
-Vulkan is a very new API, so there may be some shortcomings in the specification itself.
-You are encouraged to submit feedback to https://github.com/KhronosGroup/Vulkan-Docs[this Khronos repository].
-
-As mentioned before, the Vulkan API has a rather verbose API with many parameters to give you maximum control over the graphics hardware.
-This causes basic operations like creating a texture to take a lot of steps that have to be repeated every time.
-Therefore we'll be creating our own collection of helper functions throughout the tutorial.
-
-Every chapter will also conclude with a link to the full code listing up to that point.
-You can refer to it if you have any doubts about the structure of the code, or if you're dealing with a bug and want to compare.
-All of the code files have been tested on graphics cards from multiple vendors to verify correctness.
-Each chapter also has a comment section at the end where you can ask any questions that are relevant to the specific subject matter.
-Please specify your platform, driver version, source code, expected behavior and actual behavior to help us help you.
-
-If you have any type of feedback on the tutorial and site itself, then please don't hesitate to submit an issue or pull request to the https://github.com/KhronosGroup/Vulkan-Tutorial[GitHub repository].
-You can _watch_ the repository to be notified of updates to the tutorial.
-
-After you've gone through the ritual of drawing your very first Vulkan powered triangle onscreen, we'll start expanding the program to include linear transformations, textures and 3D models.
-
-If you've played with graphics APIs before, then you'll know that there can be a lot of steps until the first geometry shows up on the screen.
-There are many of these initial steps in Vulkan, but you'll see that each of the individual steps is easy to understand and does not feel redundant.
-It's also important to keep in mind that once you have that boring looking triangle, drawing fully textured 3D models does not take that much extra work, and each step beyond that point is much more rewarding.
-
-If you encounter any problems while following the tutorial, then first check the FAQ to see if your problem and its solution is already listed there.
-If you are still stuck after that, then feel free to ask for help in the comment section of the closest related chapter.
+All the Vulkan functions and types are linked to the specification, so you
+can click them to learn more.
+Vulkan is a very new API, so there may be some shortcomings in the
+specification itself.
+You are encouraged to submit feedback to https://github
```

Comment:
It looks like the line width changed for all adoc files. This makes it kinda hard to compare and in some cases also breaks links:

![image](https://github.com/user-attachments/assets/504e9fd1-0926-4eb7-a889-a4ef1d85c035)

Is there a way to revert the line width/break changes?

Reply:
There's not a good way to do that sadly :(  The line width in the adoc pages didn't seem to work so I just grabbed the original markdown settings and used them.